findIndex/index.js

```diff
@@ -0,0 +1,10 @@
+function findIndex(arr) {
+  const sum = arr.reduce((prev, next) => prev + next, 0);
+  let left = arr[0];
+  for (let i = 1; i < arr.length; i++) {
+    const right = sum - arr[i] - left;
+    return left === right ? i : (left += arr[i]);
```

Comment:
this will make your function to return on the first iteration always. Why are you using ternary here?